forwards_paper.tex

```diff
@@ -358,45 +376,37 @@ \subsection*{Recording the pedigree in forwards time}
     }
 \end{figure}
 
-It is significantly more efficient to generate mutations as a separate process
-after the topological simulation has completed because we will generate many
-mutations that will be lost in the population. Figure~\ref{fig:wf-trees} shows
-an example of a marginal genealogy produced by a forwards-time Wright-Fisher
-process like Algorithm~\algref{W}. There are two different trees in this
-figure: one shows all the edges output by the simulation and the other
+Besides the cost of transmitting genotypes,
+generating mutations as a separate process after the topological simulation has completed 
+avoids generating the many mutations that are lost in the population. 
+Figure~\ref{fig:wf-trees} shows
+an example of a marginal genealogy produced by a forwards-time Wright--Fisher
+process like Algorithm~\algref{W}. 
+On the left is the tree showing all the edges output by the simulation, 
+while on the right
 is the minimal tree representing the ancestry of the currently alive
 population. Clearly there is a great deal of redundancy in the topological
 information output by the simulation.
 
 There are two sources of redundancy here. The first type of redundancy arises
 from nodes in tree that have only one child. In Algorithm~\algref{W} we do
 not attempt to track coalescence events but simply record all parent-child
-relationships in the history of the population. As such, my of these edges
+relationships in the history of the population. As such, many of these edges
 will record the simple passing of genealogical information from parent to child
 and only some small subset will correspond to coalesences within the marginal
 trees. The second source of redundancy in the output of Algorithm~\algref{W}
-is due to the fact that lineages die out: there will be a large number of
-individuals in the simulation that leave no ancestors in the present day
-population. Node 26 in Figure~\ref{fig:wf-trees}a, for example, leaves no
-ancestors in the current population and so the entire path tracing back to
+is due to the fact that lineages die out: a large number of
+individuals in the simulation leave no descendants in the present day population. 
+Node 26 in Figure~\ref{fig:wf-trees}a, for example, leaves no
+ancestors in the current population, and so the entire path tracing back to
 the root is redundant.
 
-One of the reasons that it is more efficient to generate mutations on the
```

Comment:
I think we already made this point two paragraphs up?  It is a bit redundant, at least.